pycromanager/acquisitions.py

```diff
@@ -370,7 +371,7 @@ def _start_events(self, **kwargs):
 
         self._event_thread = threading.Thread(
             target=_run_acq_event_source,
-            args=(self._bridge_port, self.event_port, self._event_queue, self._bridge_timeout, self._debug),
+            args=(self._bridge_port, self._bridge_timeout, self.event_port, self._event_queue, self._debug),
```

Comment:
Looks like the order or arguments got swapped

Reply:
Yes, that was deliberate as I wanted to make the argument order consistent across all instances.  I also changed the order of the arguments in the function....
```def _run_acq_event_source(bridge_port, bridge_timeout, event_port, event_queue, debug=False):```

would be happy to change back if you prefer it.

Reply:
oh wait I see you changed the order